packages/server/api/src/app/flows/flow/flow-v2.controller.ts

```diff
@@ -0,0 +1,67 @@
+import {
```

Comment:
im no longer gonna add theses paths to be included in the mcp, they serve to test the agent functionality, they dont work well anyways for the assistant. I will remove some other endpoints aswell its cluttering the mcp.